src/lib/utils/options/readers/tsconfig.ts

```diff
@@ -26,6 +26,30 @@ export class TSConfigReader extends OptionsComponent
     private static OPTIONS_KEY:string = 'tsconfig';
 
 
+    private readConfigFile(event:DiscoverEvent, fileName:string) {
+        if (!FS.existsSync(fileName)) {
```

Comment:
Probably not if it's already handled. However, is it possible to skip most of this code and have `ts` read the entire file with `extends` instead? I thought there was a method for it.